src/test/resources/com/puppycrawl/tools/checkstyle/filters/strategy/FileTabCharacter/newline/zeroSizeConfig.xml

```diff
@@ -6,4 +6,8 @@
   <module name="FileTabCharacter">
     <property name="eachLine" value="true"/>
   </module>
+  <module name="com.puppycrawl.tools.checkstyle.filters.SuppressionPatchFilter">
+    <property name="file" value="src/test/resources/com/puppycrawl/tools/checkstyle/filters/strategy/FileTabCharacter/zeroSize.patch" />
```

Comment:
rename file to `zeroContext.patch`

https://en.wikipedia.org/wiki/Diff#Context_format

do same for other patch files.
please rename all java variables to keep `Context` in name.

Reply:
done